adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -3052,4 +3192,44 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
       loadParquetFragments(pathName, optPredicate = optPredicate, optProjection = optProjection)
     }
   }
+
+  /**
+   * Return integer size of partitions if the specified path of Parquet + Avro files is partitioned.
```

Comment:
Didn't you have a better suggestion for this doc previously?  `Return length of partitions in base pairs...` perhaps?

Reply:
fixed